### ISSUE TITLE
SelectOneMenu: regression due to today´s changes?

### ISSUE THREAD

**Issue by christophs78** [primefaces/primefaces#6866]
see https://github.com/primefaces-extensions/primefaces-integration-tests/runs/1736016093?check_suite_focus=true

```
Error:  Errors: 
Error:    SelectOneMenu001Test.testDisablePanel:104 » Javascript javascript error: Canno...
Error:    SelectOneMenu001Test.testEnablePanel:122 » Javascript javascript error: Cannot...
[INFO] 
Error:  Tests run: 236, Failures: 0, Errors: 2, Skipped: 0
```

```
2021-01-20T16:02:42.4141910Z org.openqa.selenium.JavascriptException: 
2021-01-20T16:02:42.4143850Z javascript error: Cannot read property 'off' of undefined
```

**Comment by melloware** [comment]
I think its a caching issue on GitHub Actions.  Once we put PF back to 10.0-SNAPSHOT I will try the Integration tests again.  Right now its set for 10.0-RC1 so its not building the SNAPSHOT and using an old SNAPSHOT.

**Comment by melloware** [comment]
Yep this was just a caching issue on the server.  Fixed once I put the 10.0.0 SNAPSHOT back

![image](https://user-images.githubusercontent.com/4399574/105243526-2e215080-5b3d-11eb-96da-b41dec5a8551.png)
